### ISSUE TITLE
Перенёс TODO из репозитория app112

### ISSUE THREAD

**Issue by zerobig** [egorshishkovets/Myapp112#5]
Что надо доделать:
- форму approve.html.erb. Надо написать более подробно. Что получится после нажатия на кнопки "Approve" и "Undo";
- отказаться от таблиц в оформлении. Только div и соответствующий CSS;
- отказаться от названий полей, и других текстов в оформлении. Для этого есть config/locales/ru.yml;
- Привести к общему виду форматирование программных текстов и вьюх. Отступ два пробела и между функций пустая строка;
- разобраться с реакцией системы на гостя (должен быть редирект на sign_in) и попыткой выполнить запрещённую (cancan) операцию (должен быть редирект на страницу с описанием проблемы, а не на ошибку Ruby)

Под вопросом:
- Реализовать более понятное отображение истории. Что для какой-то записи требовалось подтверждение и что запись была подтверждена или отменена и сделал это такой-то пользователь и тогда-то.

**Comment by zerobig** [comment]
Сделано из этого:
- форму approve.html.erb. Надо написать более подробно. Что получится после нажатия на кнопки "Approve" и "Undo";
- разобраться с реакцией системы на гостя (должен быть редирект на sign_in)

Частично:
- отказаться от таблиц в оформлении. Только div и соответствующий CSS;
- отказаться от названий полей, и других текстов в оформлении. Для этого есть config/locales/ru.yml;
- Привести к общему виду форматирование программных текстов и вьюх. Отступ два пробела и между функций пустая строка;

**Comment by egorshishkovets** [comment]
Просмотрел последнюю версию, но пока с ней не разбирался. Наткнулся на ошибку, не возможности посещения страницы equipmentclasses. Думаю, что это из-за того, что в seed.rb по не добавлены изображения. Две ветки пока объединять не буду

**Comment by egorshishkovets** [comment]
Объединил, то что было у меня и то то в ветке master,после объединения с новыми коммитами в ветке pdfbranch, как доделаю отчеты( внужный вид) кину в master. В общем на данный момент, последняя версия в ветке pdfbranch

**Comment by zerobig** [comment]
А у меня всё равно нет возможности новые коммиты засовывать в pdfbranch. Т.к. моя ветка сделана из master. Вот если я срисую ветку pdfbranch и начну в неё коммитить, тогда - да.
Т.е. ты предлагаешь master забросить?

**Comment by egorshishkovets** [comment]
Нет, забрасывать её не стоит) Я закинул в другую ветку для избежания
конфликтов в файлах. Но получается последних двух комитета в ней уже нет, я
подумаю как их разрешить

Вс, 13 нояб. 2016 г. в 22:50, zerobig <notifications@github.com>:

**Comment by egorshishkovets** [comment]
Сейчас объеденю свою версию с последней и залью тогда
